docs/docs/task-environment-vars.md

```diff
@@ -44,6 +44,10 @@ names through the `--env_vars_prefix` command line flag.
   was used to start this task. For example `2015-04-02T09:37:00.596Z`.
 - `MARATHON_APP_DOCKER_IMAGE` contains the value of the app definition 
   `container.docker.image` value. For example `mesosphere/marathon:latest`, `nginx` or `nginx:1.9.3`.
+- `MARATHON_APP_CPUS` contains the value of the app definition `cpus` value.
+- `MARATHON_APP_MEMORY` contains the value of the app definition `mem` value, expressed in megabytes.
+- `MARATHON_APP_DISK_SPACE`  value of the app definition `disk` value, expressed in megabytes.
+- `MARATHON_APP_LABELS` contains list of labels of the corresponding app definition. For example: `label1:value1;label2:value2`
```

Comment:
I will follow discussion here: http://stackoverflow.com/questions/2821043/allowed-characters-in-linux-environment-variable-names to provide environment variable name safety